### DISCUSSION TITLE
Scratch Addon Weekly

### DISCUSSION THREAD

**Discussion by Mbrick2** [ScratchAddons/ScratchAddons#4077] [Ideas]
Weekly Updates.

**Comment by cobaltt7** [comment]
we update whenever wl can basically. we aim for every 2 weeks.

**Comment by WorldLanguages** [comment]
Quality > quantity, right? :P

**Reply by retronbv** [reply]
garbomuffin addons > basically everything else

**Reply by Mbrick2** [reply]
I mean what has bean merged
